ipn/ipnlocal/network-lock.go

```diff
@@ -426,6 +429,15 @@ func (b *LocalBackend) NetworkLockInit(keys []tka.Key, disablementValues [][]byt
 	var ourNodeKey key.NodePublic
 	var nlPriv key.NLPrivate
 	b.mu.Lock()
+
+	// TODO(tom): Remove this guard for 1.35 and later.
+	if !envknob.UseWIPCode() && !b.capTailnetLockAlpha {
```

Comment:
time for a method instead? you have it twice.

permitTailnetLockLocked

(hah.... LockLocked)

The second Locked is for `b.mu` must be held.

Reply:
2 is the magic number? XD

Will do.

Reply:
It's fine as-is. No need for a method really. The WIP part will go away anyway, right?